### DISCUSSION TITLE
rp6502 vga branch build error: empty scalar initializer

### DISCUSSION THREAD

**Discussion by brentward** [picocomputer/community#32] [Q&A] [answered]
I'm getting an error when I try to build the current version of the vga branch.

The errors are for lines 11 and 12 in src/vga/usb/cdc/c:
https://github.com/picocomputer/rp6502/blob/f9a7ae45812e56eb89eee867e97b49305e383c0e/src/vga/usb/cdc.c#L11C1-L12C46
It says `error: empty scalar initializer`
I am able to get it to build by changing it to:
```
static absolute_time_t break_timer = {0};
static absolute_time_t faux_break_timer = {0};
```
The git blame leads me to believe it has something to do with the pseudo break for Mac but I'm not quite sure how the pseudo break is intended to be used so I can't test the change.

I'm wondering if this is an issue with my environment or will this prevent it from compiling for everyone?

**Comment by rumbledethumps** [comment, marked as answer]
The Pi Pico SDK tries to make absolute_time_t opaque with two code branches looking at NDEBUG. You're on a dev branch which I haven't done a release build on in a while.
https://github.com/picocomputer/rp6502/commit/c5ca09200472b736802d9014405405675d811a48
